aspnetcore/fundamentals/websockets.md

```diff
@@ -15,35 +16,38 @@ uid: fundamentals/websockets
+This article explains how to get started with WebSockets in ASP.NET Core. [WebSocket](https://wikipedia.org/wiki/WebSocket) ([RFC 6455](https://tools.ietf.org/html/rfc6455)) is a protocol that enables two-way persistent communication channels over TCP connections. It's used in apps that benefit from fast, real-time communication, such as chat, dashboard, and game apps.
 
 [View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/fundamentals/websockets/sample) ([how to download](xref:tutorials/index#how-to-download-a-sample)). See the [Next Steps](#next-steps) section for more information.
 
-
 ## Prerequisites
 
-* ASP.NET Core 1.1 (doesn't run on 1.0)
-* Any OS that ASP.NET Core runs on:
+* ASP.NET Core 1.1 or later
+* Any OS that supports ASP.NET Core:
   
-  * Windows 7 / Windows Server 2008 and later
+  * Windows 7 / Windows Server 2008 or later
   * Linux
   * macOS
-
-* **Exception**: If your app runs on Windows with IIS, or with WebListener, you must use:
+  
+* If the app runs on Windows with IIS:
 
   * Windows 8 / Windows Server 2012 or later
   * IIS 8 / IIS 8 Express
-  * WebSocket must be enabled in IIS
+  * WebSockets must be enabled in IIS (See the [IIS/IIS Express support](#iisiis-express-support) section.)
+  
+* If the app runs on [HTTP.sys](xref:fundamentals/servers/httpsys):
 
-* For supported browsers, see http://caniuse.com/#feat=websockets.
+  * Windows 8 / Windows Server 2012 or later
+
+* For supported browsers, see https://caniuse.com/#feat=websockets.
 
 ## When to use it
 
-Use WebSockets when you need to work directly with a socket connection. For example, you might need the best possible performance for a real-time game.
+Use WebSockets to work directly with a socket connection. For example, use WebSockets for the best possible performance with a real-time game.
 
-[ASP.NET SignalR](https://docs.microsoft.com/aspnet/signalr/overview/getting-started/introduction-to-signalr) provides a richer application model for real-time functionality, but it runs only on ASP.NET, not ASP.NET Core. A Core version of SignalR is under development; to follow its progress, see the [GitHub repository for SignalR Core](https://github.com/aspnet/SignalR).
+[ASP.NET SignalR](/aspnet/signalr/overview/getting-started/introduction-to-signalr) provides a richer app model for real-time functionality, but it only runs on ASP.NET 4.x, not ASP.NET Core. An ASP.NET Core version of SignalR is scheduled for release with ASP.NET Core 2.1. See [ASP.NET Core 2.1 high-level planning](https://github.com/aspnet/Announcements/issues/288).
 
-If you don't want to wait for SignalR Core, you can use WebSockets directly now. But you might have to develop features that SignalR would provide, such as:
+Until SignalR Core is released, WebSockets can be used. However, features that SignalR provides must be provided and supported by the developer, such as:
```

Comment:
~~...supported by the developer, such as:~~
...supported by the developer. For example: